test-suite/js/src/test/require-jdk8/org/scalajs/testsuite/javalib/math/BigIntegerConvertJSTest.scala

```diff
@@ -1,19 +0,0 @@
-package org.scalajs.testsuite.javalib.math
-
-import java.math.BigInteger
-
-import org.scalajs.jasminetest.JasmineTest
-
-object BigIntegerConvertJSTest extends JasmineTest {
-  describe("BigIntegerConvertTest") {
-    // To test that it works with strict floats, do:
-    //   > set scalaJSSemantics in testSuite ~= { _.withStrictFloats(true) }
-    when("strict-floats").
-    it("testDoubleValueZero") {
```

Comment:
Oh yes, for some reason it exists already in `BigIntegerConvertTest` without the restriction on `strict-floats`. 
https://github.com/nicolasstucki/scala-js/blob/port-test-to-junit/test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/math/BigIntegerConvertTest.scala#L190